test/Factory/AcceptListenerFactoryTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ */
+
+namespace ZFTest\Versioning\Factory;
+
+use Interop\Container\ContainerInterface;
+use PHPUnit_Framework_TestCase as TestCase;
+use ReflectionClass;
+use ZF\Versioning\AcceptListener;
+use ZF\Versioning\Factory\AcceptListenerFactory;
+
+class AcceptListenerFactoryTest extends TestCase
+{
+    public function setUp()
```

Comment:
agreed, but I think it will be a good practice keep the same visibility (and have only `test*` methods in the test case `public`) as well as call parent `setUp` method just in case we decide to have our common `TestCase` instead extending default `PHPUnit_Framework_TestCase`